hbp_validation_framework/utils.py

```diff
@@ -8,13 +8,18 @@
 Action                                 Method
 ====================================   ====================================
 View JSON data in web browser          :meth:`view_json_tree`
+Prepare test for execution             :meth:`prep_run_test`
```

Comment:
For clarity, I think this should be "prepare_run_test_offline"

Reply:
Done